app/scripts/modules/core/src/pipeline/executions/executionGroup/ExecutionGroup.tsx

```diff
@@ -276,14 +276,16 @@ export class ExecutionGroup extends React.Component<IExecutionGroupProps, IExecu
                     </span>
                   )}
                 </h4>
-                <EntityNotifications
-                  entity={pipelineConfig}
-                  application={this.props.application}
-                  entity-type="pipeline"
-                  hOffsetPercent="20%"
-                  placement="top"
-                  onUpdate={() => this.props.application.refresh()}
-                />
+                {pipelineConfig && (
```

Comment:
I'm not sure if a `strategyConfig` can have entity tags.  Seems like it could, in which case this code should pass `pipelineConfig || strategyConfig` to the `EntityNotifications` component.   

@nvndnl do you know?

Reply:
I've found the code that adds the tags to pipelines and it does not add tags to strategies.

https://github.com/spinnaker/deck/blob/7a4b0cd77d993d801f4b9b96178a8f0b4a69fd8b/app/scripts/modules/core/src/entityTag/entityTags.dataSource.ts#L24-L49